### PR TITLE
Update metar.ts

### PR DESCRIPTION
**Pull request by kjporter**
Adds density altitude calculation and display to METAR request replies

### CODE REVIEW

File: metar.ts [hunk 1]
```diff
@@ -39,6 +39,10 @@ const sendMetarMessage = async (metar: string, msg: Message) => {
 		embed.addField('Temperature :thermometer:', `${m.temperature.celsius}°C (${m.temperature.farenheit || Math.round(m.temperature.celsius * 9 / 5)}°F)`);
 		embed.addField('Dewpoint :droplet:', `${m.dewpoint.celsius}°C (${m.dewpoint.farenheit || Math.round(m.dewpoint.celsius * 9 / 5)}°F)`);
 		embed.addField('Altimeter :chart_with_upwards_trend:', `${m.altimeter.inches} inHg (QNH ${m.altimeter.millibars})`);
+		var da = calcDA(m.station,m.temperature.celsius,m.altimeter.inches);
```
Comment (arobison203) [inline metar.ts]: Please use `const` or `let` instead of `var`

File: metar.ts [hunk 2]
```diff
@@ -64,3 +68,15 @@ export default async (msg: Message) => {
 	}
 	await sendMetarMessage(metar, msg);
 };
+
+function calcDA(icao_id:string,temp:number,alt_set:number) { // Calculate density altitude for ZAB airfields
+    icao_id = icao_id.toUpperCase();
+    var elevation: {[key:string]:number} = {KAMA:3649,KCVS:4295,KROW:3671,KELP:3962,KBIF:3947,KHMN:4093,KABQ:5355,KAEG:5837,KSAF:6349,KSKX:7095,KFHU:4719,KTUS:2643,KDMA:2704,KRYN:2419,KPHX:1135,KIWA:1384,KFFZ:1394,KGYR:969,KLUF:1085,KGEU:1071,KDVT:1478,KSDL:1510,KGXF:883,KPRC:5045,KSEZ:4831,KFLG:7015,KINW:4941,KGUP:6472};
+    if(elevation.hasOwnProperty(icao_id)) {
```
Comment (arobison203) [inline metar.ts]: Never been a big fan of `Object.hasOwnProperty`, instead just preferring the truthy check `if(elevation[icao_id])`.

File: metar.ts [hunk 3]
```diff
@@ -64,3 +68,15 @@ export default async (msg: Message) => {
 	}
 	await sendMetarMessage(metar, msg);
 };
+
+function calcDA(icao_id:string,temp:number,alt_set:number) { // Calculate density altitude for ZAB airfields
+    icao_id = icao_id.toUpperCase();
```
Comment (arobison203) [inline metar.ts]: camelCase is the JS style for variables: `icaoId`

**Review by arobison203 — CHANGES_REQUESTED** [review]
Need to run eslint to format the code as well.

File: metar.ts (hunk 1 above)
Comment (kjporter) [reply metar.ts]: Done

File: metar.ts (hunk 2 above)
Comment (kjporter) [reply metar.ts]: Removed and replaced

File: metar.ts (hunk 3 above)
Comment (kjporter) [reply metar.ts]: Done

**Comment by kjporter** [comment]
Worked all identified issues

**Review by arobison203 — APPROVED** [review]

**Pull request merged**